Automattic/simplenote-ios
# Nü Note Options: Mark VI

### Details:
In this PR we're tweaking a bit the new Note Options UI:

1. Adds a new asset for the Note Options Button
2. Adds a disclosure indicator to the Collaborate row
3. Updates **Copy Link** and **Copy Internal Link** so that Note Options get dismissed on press
4. Wires few missing Tracker invocations

@eshurakov Sorry to bug you (yet again!!!) 😄 
Closes #763

### Test
1. Launch Simplenote
2. Open any note

- [x] Verify there's an ellipsis icon on the top right corner
- [x] Verify that pressing over such icon presents the Note Options
- [x] Verify the Collaborate row has a chevron `>` on its right hand side
- [x] Verify that pressing **Copy Link** dismisses the Options UI
- [x] Verify that pressing  **Copy Link** does nothing if **publish** was off
- [x] Verify that pressing **Copy Link** copies the Public Link (when Publish is On, and you've got a Public URL)

### Release
These changes do not require release notes.
